Data/Repositories/RecruitmentRepository.cs

```diff
@@ -27,27 +27,13 @@ public IQueryable<Recruitment> GetAllRecruitments()
             return result;
         }
 
-        public int AddRecruitment(Recruitment recruitment)
+        public void AddRecruitment(Recruitment recruitment)
```

Comment:
Why you need this method?

Reply:
I need it to add Recruitments, I have only changed the returned type because I have moved catching errors to the service and I will return int there based on whether I catch an error or not